src/features/git/components/ChangesFileList.test.tsx

```
@@ -0,0 +1,83 @@
+				onDiscardFile={onDiscardFile}
+				onIncludeAll={vi.fn()}
+				onIncludeNone={vi.fn()}
+				{...props}
+			/>
+		</ChakraProvider>,
+	);
+
+	return { onSelect, onOpenFile, onDiscardFile };
+}
+
+describe("ChangesFileList", () => {
```

ESLint

[error] `describe`s should begin with lowercase

(test/prefer-lowercase-title)